contracts/grpc-contracts/src/main/proto/lifeos/calendar/v1/calendar_availability.proto

```
@@ -0,0 +1,31 @@
+import "lifeos/common/v1/request_metadata.proto";
+
+option java_multiple_files = true;
+option java_outer_classname = "CalendarAvailabilityProto";
+option java_package = "com.lifeos.grpc.calendar.v1";
+
+service CalendarAvailabilityService {
+  rpc FindBusyIntervals(FindBusyIntervalsRequest) returns (FindBusyIntervalsResponse);
+}
+
+message FindBusyIntervalsRequest {
+  lifeos.common.v1.RequestMetadata metadata = 1;
```

Buf (1.72.0)

[error] cannot find `lifeos.common.v1.RequestMetadata` in this scope

(COMPILE)